### PR TITLE
Fix vscode-e2e flake

### PR DESCRIPTION
**Pull request by infiniteregrets**
closes #1518

### CODE REVIEW

File: vscode-ext/src/tests/e2e.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ describe("mirrord sample flow test", function () {
         await setBreakPoint(fileName, browser, defaultTimeout);
         await startDebugging();
 
-        const inputBox = await InputBox.create();
+        const inputBox = await InputBox.create(10000);
```
Comment (meowjesty) [inline vscode-ext/src/tests/e2e.ts:81]: What's going on here?

Comment (infiniteregrets) [reply vscode-ext/src/tests/e2e.ts:81]: we needed to extend the timeout here to 10000 since their default timeout was way too less and lead to some flakes

**Review by meowjesty — APPROVED** [review]
:bear: :+1:

**Comment by infiniteregrets** [comment]
i am not sure whats going on, i opened a pr locally and same error!!

**Pull request closed without merging**